config/install/field.storage.node.su_person_first_name.yml

```diff
@@ -2,7 +2,11 @@ langcode: en
 status: true
 dependencies:
   module:
+    - field_permissions
     - node
+third_party_settings:
+  field_permissions:
+    permission_type: public
```

Comment:
i hate the module adds this without really requiring it. can this be removed and the module dependency in the config.

Reply:
Now that I have done that a few times I just committed the thing. I too wish there was a way to have that module not do that. _FINE_ I will do it again.